src/style/modal/group-creation.less

```diff
@@ -51,25 +51,6 @@
     }
   }
 
-  &__action {
-    .button-reset-default;
-    color: var(--black);
-    font-size: @font-size-base;
-    font-weight: @font-weight-bold;
-    line-height: @line-height-lg;
-    opacity: 0.32;
-    transition: all 0.25s ease-in-out;
-
-    &:hover {
-      text-decoration: underline;
-    }
-
-    &.enabled {
-      cursor: pointer;
-      opacity: 1;
-    }
-  }
-
```

Comment:
Is this not used anymore? Has it moved somewhere?

Reply:
The button has been replaced by a ui-kit variant, no need for extra css